avatars/processors/proportions.py

```diff
@@ -7,6 +7,70 @@
 
 
 class ProportionProcessor:
+    """Processor to express numeric variables as proportion of another variable.
+
+    Arguments
+    ---------
+        variable_names:
+            variables to transform
+        reference:
+            the variable of reference
+
+    Keyword Arguments
+    -----------------
+        sum_to_one:
+            set to True to ensure the sum of the variables sum to 1 once transformed.
+            default: True
+        decimal_count:
+            the number of decimals postprocessed variables should have
+
+    Examples
+    --------
+    >>> df =  pd.DataFrame(
+    ...        {
+    ...            "variable_1": [100, 10],
+    ...            "variable_2": [10, 10],
+    ...            "variable_3": [90, 30],
+    ...        }
+    ...    )
+    >>> processor = ProportionProcessor(
+    ...    variable_names=["variable_2", "variable_3"],
+    ...    reference="variable_1",
+    ...    sum_to_one=True,
+    ... )
+    >>> preprocessed = processor.preprocess(df=df)
+
+    This processor allows you to avatarize some variable as proportion of another variable.
```

Comment:
I don't think we should use the word `"avatarize"` here, as this preprocessor is applied before avatarizing and not afer